dangerfile.ts

```diff
@@ -3,7 +3,7 @@ import { message, danger } from "danger"
 export default async () => {
   // Print out a message to the PR
   const deployURL = process.env.PR_DEPLOY_URL_ROOT
-  message(`Deployed to [a PR branch](${deployURL}) :tada:`)
+  message(`Deployed to [a PR branch](${deployURL}) :tada: - try with the [name orta](${deployURL}?name=orta).`)
```

Comment:
Note that this isn't changed, because the master version of the dangerfile.ts is used ( for security reasons )